apps/cloudbroker/base/cloudapi__machines/methodclass/cloudapi_machines.py

```diff
@@ -917,6 +921,10 @@ def create(
 
         """
         cloudspace = self.models.cloudspace.get(cloudspaceId)
+        if cloudspace.status == resourcestatus.Cloudspace.DELETED:
```

Comment:
This should be added to validatecreate which has the destroyed check and is called anyway